tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py36
+envlist = py39
```

Comment:
Python3.6 is on buildvm, let's also test with this :)

Reply:
didn't mean to include that, thanks.
my test container only has 3.9, guess i should fix that